lib/plugins/index.js

```diff
@@ -1,8 +1,13 @@
 const { pluginPingPong } = require('./ping-pong')
 const { pluginWelcomeMessage } = require('./welcome-message')
+const { pluginRandomMessage } = require('./random-messages')
```

Comment:
Ennek itt így kéne lennie:

```suggestion
const { pluginRandomMessages } = require('./random-messages')
```

hiszen az export
```js
module.exports = { pluginRandomMessages }
```

Reply:
Vazze, egyszerűen lemaradt egy karakter, észre sem vettem